### DISCUSSION TITLE
Support for DIYRuZ_Flower

### DISCUSSION THREAD

**Discussion by Anatol6868** [arendst/Tasmota#13085] [Q&A] [answered]
Will there be support for DIYRuZ_Flower? https://zigbee.blakadder.com/DIYRuZ_Flower.html

**Comment by sfromis** [comment, marked as answer]
Any reason to believe that there is no support already? It may not be "verified" but that does not make it unlikely to be supported.

**Reply by Jason2866** [reply]
A support badge is only added if feedback is provided. So missing does NOT mean not working.
Buy one and give feedback 😎

**Comment by Anatol6868** [comment]
19:41:35.254 MQT: tele/tasmota_ZB/RESULT = {"ZbState":{"Status":34,"IEEEAddr":"0x0000xxxxxxxxxxxxxxx","ShortAddr":"0xDF65","ParentNetwork":"0x0000","JoinStatus":1,"Decision":0}}
19:41:36.206 MQT: tele/tasmota_ZB/RESULT = {"ZbState":{"Status":30,"IEEEAddr":"0x00xxxxxxxxxxxxxxx","ShortAddr":"0xDF65","PowerSource":false,"ReceiveWhenIdle":false,"Security":false}}
19:41:36.387 MQT: tele/tasmota_ZB/DF65/SENSOR = {"Device":"0xDF65","0003!01":"","IdentifyQuery":true,"Endpoint":2,"LinkQuality":154}
19:41:36.521 MQT: tele/tasmota_ZB/RESULT = {"ZbState":{"Status":32,"ActiveEndpoints":["0x02","0x01"]}}
19:41:37.292 ZIG: Zigbee Devices Data saved in EEPROM (20 bytes)
19:41:39.023 MQT: tele/tasmota_ZB/RESULT = {"ZbState":{"Status":33,"Device":"0xDF65","Endpoint":"0x02","ProfileId":"0x0104","DeviceId":"0x000C","DeviceVersion":2,"InClusters":["0x0402"],"OutClusters":[]}}
19:41:40.163 MQT: tele/tasmota_ZB/RESULT = {"ZbState":{"Status":33,"Device":"0xDF65","Endpoint":"0x01","ProfileId":"0x0104","DeviceId":"0x000C","DeviceVersion":2,"InClusters":["0x0000","0x0001","0x0400","0x0402","0x0403","0x0405","0x0408"],"OutClusters":[]}}
19:41:41.559 ZIG: auto-bind `ZbBind {"Device":"0xDF65","Endpoint":2,"Cluster":"0x0402"}`
19:41:42.314 MQT: tele/tasmota_ZB/RESULT = {"ZbBind":{"Device":"0xDF65","Status":0,"StatusMessage":"SUCCESS"}}
19:41:43.564 ZIG: auto-bind `ZbSend {"Device":"0xDF65","Config":{"Temperature":{"MinInterval":30,"MaxInterval":3600,"ReportableChange":0.5}}}`
19:41:44.673 MQT: tele/tasmota_ZB/DF65/SENSOR = {"Device":"0xDF65","ConfigResponse":{},"Endpoint":2,"LinkQuality":154}
19:41:45.568 ZIG: auto-bind `ZbBind {"Device":"0xDF65","Endpoint":1,"Cluster":"0x0001"}`
19:41:45.822 MQT: tele/tasmota_ZB/RESULT = {"ZbBind":{"Device":"0xDF65","Status":0,"StatusMessage":"SUCCESS"}}
19:41:47.520 ZIG: auto-bind `ZbBind {"Device":"0xDF65","Endpoint":1,"Cluster":"0x0400"}`
ZIG: auto-bind `ZbSend {"Device":"0xDF65","Config":{"BatteryVoltage":{"MinInterval":3600,"MaxInterval":14400,"ReportableChange":0.2},"BatteryPercentage":{"MinInterval":3600,"MaxInterval":14400,"ReportableChange":5}}}`
19:41:56.927 MQT: tele/tasmota_ZB/RESULT = {"ZbState":{"Status":20,"Message":"Pairing mode disabled"}}
19:41:57.526 ZIG: auto-bind `ZbSend {"Device":"0xDF65","Config":{"Illuminance":{"MinInterval":10,"MaxInterval":3600,"ReportableChange":5}}}`
19:41:59.527 ZIG: auto-bind `ZbSend {"Device":"0xDF65","Config":{"Temperature":{"MinInterval":30,"MaxInterval":3600,"ReportableChange":0.5}}}`
19:42:01.261 ZIG: {"ZbEZSPReceived":"80004265DF"}
19:42:01.269 MQT: tele/tasmota_ZB/RESULT = {"ZbRouteError":{"ShortAddr":"0xDF65","Status":66,"StatusMessage":"MAC_INDIRECT_TIMEOUT"}}
19:42:01.562 ZIG: auto-bind `ZbSend {"Device":"0xDF65","Config":{"Pressure":{"MinInterval":30,"MaxInterval":3600,"ReportableChange":1}}}`
19:42:03.221 ZIG: {"ZbEZSPReceived":"80004265DF"}
19:42:03.229 MQT: tele/tasmota_ZB/RESULT = {"ZbRouteError":{"ShortAddr":"0xDF65","Status":66,"StatusMessage":"MAC_INDIRECT_TIMEOUT"}}
19:42:03.522 ZIG: auto-bind `ZbSend {"Device":"0xDF65","Config":{"Humidity":{"MinInterval":30,"MaxInterval":3600,"ReportableChange":1}}}`
19:42:05.226 ZIG: {"ZbEZSPReceived":"80004265DF"}
19:42:05.234 MQT: tele/tasmota_ZB/RESULT = {"ZbRouteError":{"ShortAddr":"0xDF65","Status":66,"StatusMessage":"MAC_INDIRECT_TIMEOUT"}}
19:42:06.391 MQT: tele/tasmota_ZB/DF65/SENSOR = {"Device":"0xDF65","0003!01":"","IdentifyQuery":true,"Endpoint":2,"LinkQuality":154}
19:42:06.583 MQT: tele/tasmota_ZB/DF65/SENSOR = {"Device":"0xDF65","ConfigResponse":{},"Endpoint":1,"LinkQuality":154}
19:42:06.782 MQT: tele/tasmota_ZB/DF65/SENSOR = {"Device":"0xDF65","ConfigResponse":{},"Endpoint":1,"LinkQuality":154}
19:42:06.925 MQT: tele/tasmota_ZB/RESULT = {"ZbBind":{"Device":"0xDF65","Status":0,"StatusMessage":"SUCCESS"}}

**Comment by Anatol6868** [comment]
Structure

Endpoint 1 (SIMPLE_SENSOR)
Input
0000_Basic
0001_ApplicationVersion: 3
0003_HWVersion: 1
0004_ManufacturerName: modkam.ru
0005_ModelIdentifier: DIYRuZ_Flower
0007_PowerSource: 3
4000_SWBuildID: 07/07/2021 16:18
0001_PowerConfiguration
0020_BatteryVoltage: 28
0021_BatteryPercentageRemaining: 48
0200_Custom: 6275
Output
0400_IlluminanceMeasurement
0001_PowerConfiguration
0402_TemperatureMeasurement
0403_PressureMeasurement
0405_RelativeHumidityMeasurement
0408_SoilMoisture
0019_OtaUpgrade
Endpoint 2 (SIMPLE_SENSOR)
Output
0402_TemperatureMeasurement
